physionet-django/physionet/settings/base.py

```diff
@@ -500,3 +500,8 @@ class StorageTypes:
 
 SITE_NAME = config('SITE_NAME')
 EMAIL_SIGNATURE = config('EMAIL_SIGNATURE')
+
+ENABLE_RESEARCH_ENVIRONMENTS = config('ENABLE_RESEARCH_ENVIRONMENT', default=False, cast=bool)
```

Comment:
```suggestion
ENABLE_RESEARCH_ENVIRONMENTS = config('ENABLE_RESEARCH_ENVIRONMENTS', default=False, cast=bool)
```
To match what's in `.env.example` and the Python var.